src/Engine/ProtoCore/Parser/Parser.cs

```diff
@@ -4510,139 +4510,139 @@ public class Errors {
+			case 93: s = Properties.Resources.InvalidAssociativeBinaryOps; break;
+			case 94: s = Properties.Resources.InvalidAssociativeAddOp; break;
+			case 95: s = Properties.Resources.InvalidAssociativeMulOp; break;
+			case 96: s = Properties.Resources.InvalidAssociativeComparisonOp; break;
+			case 97: s = Properties.Resources.InvalidAssociativeLogicalOp; break;
+			case 98: s = Properties.Resources.InvalidAssociativeDecoratedIdentifier; break;
+			case 99: s = Properties.Resources.InvalidAssociativeUnaryExpression; break;
+			case 100: s = Properties.Resources.InvalidAssociativeUnaryop; break;
+            case 101: s = Properties.Resources.InvalidAssociativeFactor; break;
+			case 102: s = Properties.Resources.InvalidAssociativeNegop; break;
+			case 103: s = Properties.Resources.InvalidAssociativeBitOp; break;
+			case 104: s = Properties.Resources.InvalidAssociativePostFixOp; break;
+			case 105: s = Properties.Resources.InvalidAssociativeNumber; break;
+			case 106: s = Properties.Resources.InvalidAssociativeNameReference; break;
+			case 107: s = Properties.Resources.InvalidAssociativeNameReference; break;
+			case 108: s = Properties.Resources.InvalidAssociativeNameReference; break;
+			case 109: s = Properties.Resources.InvalidImperativeStmt;break;
+			case 110: s = Properties.Resources.InvalidAssociativeAccessFunctiondecl; break;
+			case 111: s = Properties.Resources.InvalidImperativeLanguageblock; break;
+			case 112: s = Properties.Resources.InvalidImperativeLanguageblock; break;
+			case 113: s = Properties.Resources.InvalidImperativeLanguageblock; break;
+			case 114: s = Properties.Resources.InvalidImperativeLanguageblock; break;
+			case 115: s = Properties.Resources.InvalidImperativeIfstmt; break;
+			case 116: s = Properties.Resources.InvalidImperativeIfstmt; break;
+			case 117: s = Properties.Resources.InvalidImperativeIfstmt; break;
+			case 118: s = Properties.Resources.InvalidImperativeForLoop; break;
+			case 119: s = Properties.Resources.InvalidImperativeAssignstmt; break;
+			case 120: s = Properties.Resources.InvalidImperativeAssignstmt; break;
+			case 121: s = Properties.Resources.InvalidImperativeDecoratedIdentifier; break;
+			case 122: s = Properties.Resources.InvalidImperativeNameReference; break;
+			case 123: s = Properties.Resources.InvalidImperativeUnaryexpr; break;
+			case 124: s = Properties.Resources.InvalidImperativeUnaryop; break;
+			case 125: s = Properties.Resources.InvalidImperativeFactor; break;
+			case 126: s = Properties.Resources.InvalidImperativeLogicallop; break;
+			case 127: s = Properties.Resources.InvalidImperativeRelop; break;
+			case 128: s = Properties.Resources.InvalidImperativeAddop; break;
+			case 129: s = Properties.Resources.InvalidImperativeMulop; break;
+			case 130: s = Properties.Resources.InvalidImperativeBitop; break;
+			case 131: s = Properties.Resources.InvalidImperativeNum; break;
+			case 132: s = Properties.Resources.InvalidImperativePostFixOp; break;
```

Comment:
@junmendoza do these changes go directly in `Parser.cs` or in one of the `.atg` files?